scripts/process-semgrep-results.js

```
@@ -0,0 +1,51 @@
+// process-semgrep-results.js
+const fs = require('fs');
```

ESLint

[error] Require statement not part of import statement.

(@typescript-eslint/no-var-requires)